standards/core/rendezvous.md

```diff
@@ -0,0 +1,64 @@
+---
+title: WAKU-RENDEZVOUS
+name: Waku rendez vous discovery
+editor: Simon-Pierre Vivier <simvivier@status.im>
+contributors:
+---
+
+## Abstract
+This document describe the goal,
+strategy and usage of the libp2p rendezvous protocol by Waku.
+
+Rendezvous is one of the discovery methods that can be used by Waku,
+it supplements Discovery v5 and Waku peer exchange but fill a specific role.
+
+## Background and Rationale
+How do new nodes join the network is the question that discovery answers.
+Discovery must be fast, pertinent and resilient to attacks.
+Rendezvous is both fast and allow the discovery of relevant peers,
+although by it's design can be easily abused.
```

Comment:
```suggestion
although it can be easily abused due to its design that clusters discoverable nodes around rendezvous points.
```

Reply:
I was thinking of DDOS

Reply:
Indeed! I guess my point is simply to qualify the statement somewhat.